test/utils.js

```diff
@@ -33,38 +32,25 @@ class ContributionReward extends ExtendTruffleContract(
 
 describe("ExtendTruffleContract", () => {
   it("Must have sane inheritance", async () => {
-    let x;
+    let scheme;
 
-    x = await ContributionReward.new();
-    assert.isOk(x.nativeToken());
-    assert.equal(x.foo(), "bar");
-    assert.equal(x.proposeContributionReward(), "abc");
-    assert.equal(await x.nativeToken(), await x.contract.nativeToken());
+    scheme = await ContributionReward.new();
+    assert.equal(scheme.foo(), "bar");
+    assert.equal(scheme.proposeContributionReward(), "abc");
     assert.equal(
-      await x.setParams({
+      await scheme.setParams({
         orgNativeTokenFee: 0,
-        schemeNativeTokenFee: 0,
         voteParametersHash: helpers.SOME_HASH,
         votingMachine: helpers.SOME_ADDRESS
       }),
-      "0xb6660b30e997e8e19cd58699fbf81c41450f200dbcb9f6a85c07b08483c86ee9"
+      "0x59af66fa0cefc060220aa22b6d9420988e6037221ca060f3140baa53883138ba"
```

Comment:
what is this address ?

Reply:
It is the hash value for the given parameters

Reply:
this ok @orenyodfat ?

Reply:
@dkent600 what this test is testing ?
why it re define ContributionReward class ?

Reply:
test ExtendTruffleContract ?

Reply:
Yes, the whole idea is to test the ablity to subclass ExtendTruffleContract .  I'll rename it so it doesn't cause so much confusion.